### DISCUSSION TITLE
Issue with hosting an app on vercel

### DISCUSSION THREAD

**Discussion by FREDVUNI** [vercel/next.js#63054] [Help] [answered]
### Summary

Any idea how to get rid of this error?

![image](https://github.com/vercel/next.js/assets/41730664/3b6363fe-5e6a-4065-8998-098d0dec1f18)

Locally, everything looks great, hosting on vercel has become complicated

thanks

**Comment by mutasim77** [comment, marked as answer]
the error is clear enough to google and find: https://github.com/prisma/prisma/discussions/11929

**Reply by FREDVUNI** [reply]
Thanks, that was it, The Ip address
